src/main/java/org/recordrobotics/charger/subsystems/Claw.java

```diff
@@ -0,0 +1,36 @@
+package org.recordrobotics.charger.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import org.recordrobotics.charger.RobotMap;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class Claw extends SubsystemBase {
+	private static final double GEAR_RATIO = 63;
```

Comment:
suspicious number, but as long as it's correct ig

Reply:
Claw group told us it was 1:63